app/controllers/api/api_controller.rb

```diff
@@ -2,13 +2,14 @@ module Api
   class ApiController < Api::BaseController
     def index
       res = {
-        :name        => Settings.base.name,
-        :description => Settings.base.description,
-        :version     => Settings.base.version,
-        :versions    => entrypoint_versions,
-        :settings    => user_settings,
-        :identity    => auth_identity,
-        :server_info => server_info,
+        :name                => Settings.base.name,
+        :description         => Settings.base.description,
+        :version             => Settings.base.version,
+        :versions            => entrypoint_versions,
+        :settings            => user_settings,
+        :identity            => auth_identity,
+        :server_info         => server_info,
+        :productization_info => productization_info
```

Comment:
I agree with @bdunne ... this should be `product_info`.  The API should expose a concept.  The fact that it's productized is not really important.  So the concept here is information about the product. Although productization happens to change the underlying values, it's still the product's information